state/execution.go

```diff
@@ -277,6 +277,8 @@ func (blockExec *BlockExecutor) ApplyBlock(
 		return state, 0, fmt.Errorf("commit failed for application: %v", err)
 	}
 
+	blockExec.logger.Info("state updated in ApplyBlock", "height", block.Height, "app version in the header", block.Header.Version.App, "updated state timeout_commit", state.TimeoutCommit, "updated state timeout_propose", state.TimeoutPropose)
```

Comment:
Do we need this log entry? We already have `executed block` and `committed state`

Reply:
removed in https://github.com/celestiaorg/celestia-core/pull/1494/commits/081ecce8d37daf491721798a672734b8692d081d